html/changelogs/fluffyghost.yml

```
@@ -0,0 +1,41 @@
+################################
+# Example Changelog File
+#
+# Note: This file, and files beginning with ".", and files that don't end in ".yml" will not be read. If you change this file, you will look really dumb.
+#
+# Your changelog will be merged with a master changelog. (New stuff added only, and only on the date entry for the day it was merged.)
+# When it is, any changes listed below will disappear.
+#
+# Valid Prefixes:
+#   bugfix
+#   wip (For works in progress)
+#   tweak
+#   soundadd
+#   sounddel
+#   rscadd (general adding of nice things)
+#   rscdel (general deleting of nice things)
+#   imageadd
+#   imagedel
+#   maptweak
+#   spellcheck (typo fixes)
+#   experiment
+#   balance
+#   admin
+#   backend
+#   security
+#   refactor
+#################################
+
+# Your name.
+author: FluffyGhost
+
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - rscadd: "Expanded the Intrepid helm room and moved the sensors console, added a window to allow the sensors console operator to see the people buckled and see the helm, added two softsuit dispensers to fill the space a little, made the window doors towards the cannon station a double window and duplicated it to the right of the sensors/helm area for symmetry."
```

```suggestion
  - maptweak: "Expanded the Intrepid helm room and moved the sensors console, added a window to allow the sensors console operator to see the people buckled and see the helm, added two softsuit dispensers to fill the space a little, made the window doors towards the cannon station a double window and duplicated it to the right of the sensors/helm area for symmetry."
```
the changelog entry is very long. consider making it more succinct, if possible.